### ISSUE TITLE
Saving programming to channels hits 500: FST_ERR_FAILED_ERROR_SERIALIZATION

### ISSUE THREAD

**Issue by markdavella** [chrisbenincasa/tunarr#400]
Repro:
1.) Add programming to channel ( I was testing adding Bluey Collection, all 3 seasons)
2.) Save Channel

Results:
Hit 500 "Failed to serialize an error.  Error: Response doesn't match the schema."

```
{
    "statusCode": 500,
    "code": "FST_ERR_FAILED_ERROR_SERIALIZATION",
    "message": "Failed to serialize an error. Error: Response doesn't match the schema. Original error: insert into `program` (`uuid`, `created_at`, `updated_at`, `source_type`, `original_air_date`, `duration`, `episode`, `file_path`, `external_source_id`, `external_key`, `plex_rating_key`, `plex_file_path`, `parent_external_key`, `grandparent_external_key`, `rating`, `season_number`, `show_icon`, `show_title`, `summary`, `title`, `type`, `year`) values ('4284a632-b45a-4a0c-a09f-026ac9dfe031', 1714409641245, 1714409641245, 'plex', '2018-10-01', 437984, 1, '/media/tv/Bluey (2018) {imdb-tt7678620}/Bluey (2018) - S01E01 - Magic Xylophone [WEBDL-1080p][EAC3 5.1][h264]-FLUX.mkv', 'ATN', '4220', '4220', '/library/parts/10497/1623572728/file.mkv', '4219', '4218', 'TV-Y', 1, '/library/metadata/4218/thumb/1714198159', 'Bluey', 'As Bluey and Bingo squabble over their magic xylophone (that has the power to freeze Dad in space and time) Dad seizes control and freezes Bluey, leaving Bingo as her only hope.', 'The Magic Xylophone', 'episode', 2018), ('ce6da626-267f-47f4-886e-ea522355f3bc', 1714409641246, 1714409641246, 'plex', '2018-10-02', 437984, 2, '/media/tv/Bluey (2018) {imdb-tt7678620}/Bluey (2018) - S01E02 - Hospital [WEBDL-1080p][EAC3 5.1][h264]-FLUX.mkv', 'ATN', '4221', '4221', '/library/parts/10498/1623572743/file.mkv', '4219', '4218', 'TV-Y', 1, '/library/metadata/4218/thumb/1714198159', 'Bluey', 'Doctor Bluey is needed when Dad gets a very curious x-ray from Nurse Bingo. It seems he has a cat in his tummy, leaving Bluey no choice but to operate immediately!', 'Hospital', 'episode', 2018), ('434fc166-a25e-4565-98e7-92a95e062efb', 1714409641246, 1714409641246, 'plex', '2018-10-03', 437984, 3, '/media/tv/Bluey (2018) {imdb-tt7678620}/Bluey (2018) - S01E03 - Keepy Uppy [WEBDL-1080p][EAC3 5.1][h264]-FLUX.mkv', 'ATN', '4222', '4222', '/library/parts/10499/1623573030/file.mkv', '4219', '4218', 'TV-Y', 1, '/library/metadata/4218/thumb/1714198159', 'Bluey', 'Bluey, Bingo and Mum are enjoying playing Keepy-Uppy around the house with their last balloon, when Dad joins in and the game becomes the ultimate test of skill.', 'Keepy Uppy', 'episode', 2018), ('ea2fbaa7-79cd-4687-aafd-47107d9db98e', 1714409641246, 1714409641246, 'plex', '2018-10-04', 437984, 4, '/media/tv/Bluey (2018) {imdb-tt7678620}/Bluey (2018) - S01E04 - Daddy Robot [WEBDL-1080p][EAC3 5.1][h264]-FLUX.mkv', 'ATN', '4223', '4223', '/library/parts/10500/1623572966/file.mkv', '4219', '4218', 'TV-Y', 1, '/library/metadata/4218/thumb/1714198159', 'Bluey', 'Bluey and Bingo don''t want to tidy the playroom, so they get Daddy Robot to do it for them. But when Bluey takes things too far, Daddy Robot malfunctions with disastrous results.', 'Daddy Robot', 'episode', 2018), ('9d87868d-0f5a-49be-b42e-265b1ddbb1be', 1714409641246, 1714409641246, 'plex', '2018-10-05', 437984, 5, '/media/tv/Bluey (2018) {imdb-tt7678620}/Bluey (2018) - S01E05 - Shadowlands [WEBDL-1080p][EAC3 5.1][h264]-FLUX.mkv', 'ATN', '4224', '4224', '/library/parts/10501/1623572971/file.mkv', '4219', '4218', 'TV-Y', 1, '/library/metadata/4218/thumb/1714198159', 'Bluey', 'On a picnic in the park, Bluey and her friends play Shadowlands. But when Coco keeps taking shortcuts, Bluey has to show her that following the rules can make the game more fun.', 'Shadowlands', 'episode', 2018), ('ca0039f6-f5b5-463c-b69b-808f957240cb', 1714409641246, 1714409641246, 'plex', '2018-10-06', 437984, 6, '/media/tv/Bluey (2018) {imdb-tt7678620}/Bluey (2018) - S01E06 - The Weekend [WEBDL-1080p][EAC3 5.1][h264]-FLUX.mkv', 'ATN', '4225', '4225', '/library/parts/10502/1623572596/file.mkv', '4219', '4218', 'TV-Y', 1, '/library/metadata/4218/thumb/1714198159', 'Bluey', 'At the weekend, Bluey and Bingo are playing in the garden with Dad. But Bingo gets upset, when she discovers a walking leaf insect and Dad is too caught up in the game to notice.', 'The Weekend', 'episode', 2018),  ... - UNIQUE constraint failed: program.source_type, program.external_source_id, program.external_key"
}
```

Note: Trimmed error messaging so I could post in GH

**Comment by chrisbenincasa** [comment]
NVM - I got it - it happens when updating an existing program in the DB

**Comment by chrisbenincasa** [comment]
Should be fixed with #401

**Comment by kpupp** [comment]
I am also getting lots of errors like this. The channels play, but the guides don't load and return a 500. If I leave them be long enough, I think it ends up borking the whole container in some funny ways. 
```
2025-08-09T00:17:24.994Z [error]: GET /api/channels/all/lineups 
    err: {
      "type": "ResponseSerializationError",
      "message": "Response doesn't match the schema",
      "stack":
          FastifyError: Response doesn't match the schema
              at new ResponseSerializationError (/snapshot/dist/bundle.cjs:1298:702)
              at /snapshot/dist/bundle.cjs:1298:478
              at serialize (/snapshot/dist/bundle.cjs:581:63794)
              at preSerializationHookEnd (/snapshot/dist/bundle.cjs:581:59091)
              at preSerializationHook (/snapshot/dist/bundle.cjs:581:58909)
              at Oi.send (/snapshot/dist/bundle.cjs:581:55566)
              at Object.<anonymous> (/snapshot/dist/bundle.cjs:1281:15573)
              at process.processTicksAndRejections (node:internal/process/task_queues:105:5)
      "code": "FST_ERR_RESPONSE_SERIALIZATION",
      "name": "FastifyError",
      "statusCode": 500,
      "method": "GET",
      "url": "/api/channels/all/lineups",
      "serialization": {
        "url": "/api/channels/all/lineups",
        "method": "GET"
      }
    }
```
